interfaces/Portal/src/app/pages/peoples-affected/peoples-affected.page.html

```diff
@@ -39,12 +20,15 @@
       </ion-col>
     </ion-row>
   </ion-grid>
+</div>
 
+<ion-content class="ion-padding">
   <app-program-people-affected
     #table
     *appIfPermissions="[Permission.RegistrationREAD]"
     [programId]="programId"
     [thisPhase]="thisPhase"
     (isCompleted)="onReady($event)"
+    displayImportRegistration="true"
```

Comment:
```suggestion
    [displayImportRegistration]="true"
```

I think the type-checking isn't enabled yet on these templates, but this would make it checkable in the future...